src/Main.java

```diff
@@ -1,6 +1,120 @@
 public class Main {
     public static void main(String[] args)
     {
-        System.out.println("Hello world!");
+        //Begin HomeWork3
+        //Задание 1:
+        byte a = 1;
+        short b = 20000;
+        int c = 50000;
+        long d = 1000000000L;
+        float e = 1.12f;
+        double f = 2.123456789;
+            System.out.println("Значение переменной a с типом byte равно " + a);
+            System.out.println("Значение переменной b с типом short равно " + b);
+            System.out.println("Значение переменной c с типом int равно " + c);
+            System.out.println("Значение переменной d с типом long равно " + d);
+            System.out.println("Значение переменной e с типом float равно " + e);
+            System.out.println("Значение переменной f с типом double равно " + f);
+
+        //Задание 2:
+        double a1 = 27.12;
+        long b1 = 987_678_965_549L;
+        float c1 = 2.786f;
+        short d1 = 569;
+        short e1 = -159;
+        int f1 = 27897;
+        byte g1 = 67;
+        System.out.println("Значение переменной a1 равно " + a1);
+        System.out.println("Значение переменной b1 равно " + b1);
+        System.out.println("Значение переменной c1 равно " + c1);
+        System.out.println("Значение переменной d1 равно " + d1);
+        System.out.println("Значение переменной e1 равно " + e1);
+        System.out.println("Значение переменной f1 равно " + f1);
+        System.out.println("Значение переменной g1 равно " + g1);
+
+        //Задание 3:
+        byte studentsL = 23;
+        byte studentsA = 27;
+        byte studentsE = 30;
+        short sheetTotal = 480;
+        short sheetPerStudent = (short) (sheetTotal / (studentsL + studentsA + studentsE));
+        System.out.println("На каждого ученика рассчитано " + sheetPerStudent + " листов бумаги");
+
+        //Задание 4:
+        byte bottle = 16;
+        byte time = 2;
+        byte efficiencyPerMin = (byte) (bottle / time);
+        short efficiencyPer20 = (short) (efficiencyPerMin * 20);
+        short efficiencyPerDay = (short) (efficiencyPerMin * 24 * 60);
+        int efficiencyPer3Day = efficiencyPerMin * 3 * 24 * 60;
```

Comment:
```suggestion
        int efficiencyPer3Day = 3 * efficiencyPerDay;
```
зачем снова считать одно и то же?